tests/layer_tests/requirements.txt

```diff
@@ -2,4 +2,5 @@ requests>=2.25.1
 numpy>=1.19.2
 torch
 torchvision
+paddlepaddle
```

Comment:
should we separate requirements per framework? 
Otherwise we easily have a situation when all requirements cannot be satisfied

CC @p-wysocki

Reply:
I think that currently there's no concrete need for separating the requirements in CI per frontend. As seen in https://github.com/openvinotoolkit/openvino/pull/15343 most of the frontends have package versions which are compatible with each other - the way you can see it, if a package version is compatible between requirements files, it is specified in a common `constraints.txt` file.

The only (kind of) incompatible package is I guess `numpy`, but it hasn't caused any issues on our CI yet, so I wouldn't rely on it when making decision about splitting requirements quickly.

However, as future proofing, I think that's a good idea. I don't think there's a rush though.

Reply:
> If we split requirements for paddle/pytorch should we also split this step https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux.yml#L534-L537 for paddle/pytorch. What do you think?

I don't mind